### DISCUSSION TITLE
Does the front-end part supports pre-rendering?

### DISCUSSION THREAD

**Discussion by xperiandri** [requarks/wiki#4794] [Help / Questions] [answered]
I wonder if the solution supports pre-rendering and it is a good idea to host the front-end part on the same server as the back-end part?

**Comment by NGPixel** [comment, marked as answer]
No prerendering is needed as page contents are already cached on the fly and any non-html content is already "prerendered".

**Reply by xperiandri** [reply]
So I can publish front-end to Azure Static Websites on BLOB storage, can't I?

**Reply by NGPixel** [reply]
No, that's not how Wiki.js works.
